NearbyNetwork/NearbyNetwork/Sources/NearbyNetworkService.swift

```diff
@@ -141,6 +139,29 @@ extension NearbyNetworkService: NearbyNetworkInterface {
             }
         }
     }
+
+    public func send(
+        fileURL: URL,
+        info: DataSource.DataInformationDTO,
+        to connection: NetworkConnection
+    ) async {
+        let infoJsonData = try? JSONEncoder().encode(info)
+
+        guard
+            let infoJsonData,
+            let infoJsonString = String(data: infoJsonData, encoding: .utf8),
+            let peer = session
+                .connectedPeers
+                .first(where: { $0.displayName == connection.id.uuidString})
```

Comment:
```suggestion
                .first(where: { $0.displayName == connection.id.uuidString })
```

띄어쓰기 ~ 안해두 린트 에러 안나나유 ? ??

Reply:
에러는 안나지만, 띄는 것이 맞는 컨벤션이어서 050abc47421d0b523ae2b69f0ac455a889718a29 에서 반영했습니다!